pkg/datapath/linux/probes/probes.go

```diff
@@ -131,11 +131,17 @@ type MapTypes struct {
 	HaveStackMapType               bool `json:"have_stack_map_type"`
 }
 
+// Misc contains bools exposing miscellaneous eBPF features.
```

Comment:
I think it would be redundant as that's already in the comment of the `Features` struct (which contains `Misc`)

https://github.com/cilium/cilium/blob/10d2ea7699492a14ca98d4424eb5a946077efdb5/pkg/datapath/linux/probes/probes.go#L139-L145